src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -161,51 +166,89 @@ protected function build_data(): void {
 
 			$this->tickets_by_type[ $ticket->type ][] = $ticket_data;
 		}
+		
+		$this->build_order_sales_data();
+	}
+	
+	/**
+	 * Build the order sales data.
+	 *
+	 * @since TBD
+	 */
+	protected function build_order_sales_data() {
+		$args = [
+			'events' => $this->get_post_id(),
+			'status' => 'any',
+		];
+		
+		$orders = tec_tc_orders()->by_args( $args )->all();
+		
+		foreach ( $orders as $order ) {
+			foreach ( $order->items as $ticket_id => $item ) {
+				$this->process_order_sales_data( $order->status_slug, $ticket_id, $item );
+			}
+		}
```

Comment:
Uff... this feels expensive. But if we have no way around it... I wonder if this should be a intentional trigger tho.

Reply:
True, We don't have any workaround to fetch the data faster but we can try to add some caching to reduce invoking it unless necessary.